### ISSUE TITLE
Better error when using data with less than 200 frames for mcorr

### ISSUE THREAD

**Issue by kushalkolar** [flatironinstitute/CaImAn#1036]
Instead of current numpy indexerror

**Comment by pgunn** [comment]
Yup, we've talked about doing this a few times and it's on the radar. But it's good to have an issue for it.

**Comment by kushalkolar** [comment]
I think it's actually 1000 frames?

**Comment by EricThomson** [comment]
I have been trying to reproduce this, and have movies of length 20, 50, 100 and have not been able to reproduce. I'm not sure if there have been changes in the code in the past year or what, but have not been able to generate the `IndexError` (e.g., as in Issue #1047 ). We should probably just have these sample movies on hand for people so they can create a minimal reproducible example with one of them to demo the problem, so we can reproduce and handle it. 

I think we should keep this issue open.  I am not sure what I'm doing wrong I'm getting good motion correction with the 20-frame movie 😄
